ci/unit_tests/compare_dialogs/main_test.py

```diff
@@ -0,0 +1,150 @@
+
+import os, pytest, json
+
+import compare_dialogs
+from .. import unit_utils
+
+class TestMain(unit_utils.TestCaseCapture):
+
+    dataBasePath = './ci/unit_tests/compare_dialogs/compare_dialogs_data/'
+
+    complexDictDifferentEmptyJsonPath = os.path.abspath(os.path.join(dataBasePath, 'complex_dict_different_empty.json'))
+    complexDictDifferentKeyJsonPath = os.path.abspath(os.path.join(dataBasePath, 'complex_dict_different_key.json'))
+    complexDictDifferentValueJsonPath = os.path.abspath(os.path.join(dataBasePath, 'complex_dict_different_value.json'))
+    complexDictJsonPath = os.path.abspath(os.path.join(dataBasePath, 'complex_dict.json'))
+    complexDictSubJsonPath = os.path.abspath(os.path.join(dataBasePath, 'complex_dict_sub.json'))
+    complexDictUnsortedJsonPath = os.path.abspath(os.path.join(dataBasePath, 'complex_dict_unsorted.json'))
+    emptyDictJsonPath = os.path.abspath(os.path.join(dataBasePath, 'empty_dict.json'))
+    emptyListJsonPath = os.path.abspath(os.path.join(dataBasePath, 'empty_list.json'))
+    noJsonPath = os.path.abspath(os.path.join(dataBasePath, 'no.json'))
+    nullJsonPath = os.path.abspath(os.path.join(dataBasePath, 'null.json'))
+    valueJsonPath = os.path.abspath(os.path.join(dataBasePath, 'value.json'))
+
+    def callfunc(self, *args, **kwargs):
+        compare_dialogs.main(*args, **kwargs)
+
+    ''' Tests some basic sets of args '''
```

Comment:
The correct docstring is on the first line of method (right after the first command, but after function header). I apologize because I had this bad format in my unit test example (but it's fixed).

Reply:
Fixed